apps/web/src/server/views/applications/case-timetable/components/timetable-items-list.component.njk

```diff
@@ -8,7 +8,11 @@
 		{% set accordionItemsPreExamination = [] %}
 
 		{% for timetableItem in timetableItems %}
-			{% set heading = [timetableItem.date if timetableItem.date else timetableItem.startDate, ' - ', timetableItem.name] | join %}
+			{% if (timetableItem.itemTypeName == 'Procedural Deadline (Pre-Examination)' or timetableItem.itemTypeName == 'Deadline') %}
```

Comment:
1) there is another one "deadline" item that should be included in this condition "Deadline for close of examination".

2) to avoid using these long names full of parenthesis you could use the field named "templateType".

<img width="429" alt="immagine" src="https://github.com/Planning-Inspectorate/back-office/assets/101253567/e8b8400a-9ab4-44e1-9f48-126d99d7e969">


the if condition would then become
if(templateType === 'deadline-startdate-mandatory' or templateType === 'deadline-startdate-mandatory')
and it would include any deadline type.

3) Feel free, if you want, to move this logic in the controller

Reply:
Have updated. Is a better solution too since both are of type 'deadline'
<img width="1097" alt="image" src="https://github.com/Planning-Inspectorate/back-office/assets/352844/de246a12-5100-4b33-871e-e8e681511bb0">